tests/test_image.py

```diff
@@ -104,6 +104,15 @@
             "docker.io",
             pytest.raises(exc.InvalidImageFormatError),
         ),
+        (
+            "docker.io/library/image:Tag",
+            "image",
+            "Tag",
+            None,
+            None,
```

Comment:
```suggestion
            "library",
```
This needs to be the repository value which is "library" in your case, as given in line 8 though I admit the description is a bit confusing

Reply:
I was just copying the previous test ("docker.io/Library/image:tag") and changed the test case.
I also noticed that the repository was set to "None" but reasoned that any Image under docker.io/library can be called by it's short name: 
```
docker pull docker.io/library/nginx
docker pull library/nginx
docker pull nginx
```
Let me know if you want me to change the repository to "library"

Reply:
I made the suggested change.

Reply:
I see what you mean. That is, however, only a unit test that does not take the docker.io specifics into account. I agree that specifically choosing `library` as a repo name in the previous example is confusing though.

thanks, looks good now.